server/model.js

```diff
@@ -8,5 +8,4 @@ const Card = mongoose.model('Card', {
   email: String,
   picture: String
 })
```

Comment:
may be you can create a folder by name "models" and keep card.js inside , you can jerry's folder